KDOP_tree/include/CGAL/KDOP_tree/internal/Has_nested_type_Shared_data.h

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2019  University of Cambridge (UK), GeometryFactory (France)
```

Comment:
There is no need for this file as nothing is kdop specific.  Just include the corresponding file from the AABB_tree package.

Reply:
This file has been removed and replaced by the corresponding AABB_tree file.